mava/systems/rec_mappo.py

```diff
@@ -584,6 +601,14 @@ def run_experiment(_config: DictConfig) -> None:
     """Runs experiment."""
     config = copy.deepcopy(_config)
 
+    # Set recurrent chunk size.
```

Comment:
Just a thought, should we require the user to set it so we don't have to have this check? I don't mind either way really

Reply:
I think what we have at the moment is quite nice because it is easy to recover the current behaviour.